src/types.js

```diff
@@ -29,6 +29,11 @@ export type IterationKind = "key+value" | "value" | "key";
 export type SourceType = "module" | "script";
 
 export type Compatibility = "browser" | "jsc-600-1-4-17" | "node-source-maps";
+export const CompatibilityValues = {
+  BROWSER: "browser",
+  MOBILE_JSC: "jsc-600-1-4-17",
+  NODE_SOURCE_MAPS: "node-source-maps",
+};
```

Comment:
OK! I've just notices that `jsc-600-1-4-17` is something that you probably don't want to repeat a lot.